src/rulesets/Base/array.jl

```diff
@@ -166,18 +166,19 @@ function frule((_, ẋs), ::typeof(repeat), xs::AbstractArray, cnt...; kw...)
     return repeat(xs, cnt...; kw...), repeat(ẋs, cnt...; kw...)
 end
 
-function rrule(::typeof(repeat), xs::AbstractArray; inner=ntuple(Returns(1), ndims(xs)), outer=ntuple(Returns(1), ndims(xs)))
+function rrule(::typeof(repeat), xs::AbstractArray; inner=nothing, outer=nothing)
```

Comment:
this brings it in line with https://github.com/JuliaLang/julia/blob/master/base/abstractarraymath.jl#L392